### PR TITLE
Reports issues and PRs without release note.

### PR DESCRIPTION
**Pull request by artursouza**
# Description

Reports issues and PRs without release note.

## Issue reference

We strive to have all PR being opened based on an issue, where the problem or feature have been discussed prior to implementation.

Please reference the issue this PR will close: Endgame

## Checklist

Please make sure you've  completed the relevant tasks for this PR, out of the following list:

* [ ] Code compiles correctly
* [ ] Created/updated tests
* [ ] Unit tests passing
* [ ] End-to-end tests passing
* [ ] Extended the documentation / Created issue in the https://github.com/dapr/docs/ repo: dapr/docs#_[issue number]_
* [ ] Specification has been updated / Created issue in the https://github.com/dapr/docs/ repo: dapr/docs#_[issue number]_
* [ ] Provided sample for the feature / Created issue in the https://github.com/dapr/docs/ repo: dapr/docs#_[issue number]_

RELEASE NOTE: NOT APPLICABLE

### CODE REVIEW

**Review by wcs1only — DISMISSED** [review]

File: .github/scripts/generate_release_notes.py [hunk 1]
```diff
@@ -108,22 +109,27 @@ def get_repo_priority(name):
...
         a = [l.login for l in issueOrPR.assignees]
         contributors.extend(a)
-        url = issueOrPR.html_url
     match = re.search(releaseNoteRegex, issueOrPR.body, re.M)
+    hasNote = False
     if match:
         repo = issueOrPR.repository
         note = match.group(1).strip()
-        if note:
+        if note and (note.upper() != "NOT APPLICABLE"):
```
Comment (artursouza) [inline .github/scripts/generate_release_notes.py:125]: There is a bug here. Fixing.

File: .github/scripts/generate_release_notes.py [hunk 2]
```diff
@@ -108,22 +109,27 @@ def get_repo_priority(name):
...
-    url = ""
+    url = issueOrPR.html_url
     try:
         # only a PR can be converted to a PR object, otherwise will throw error.
         pr = issueOrPR.as_pull_request()
         contributors.append(pr.user.login)
-        url = pr.html_url
     except:
         a = [l.login for l in issueOrPR.assignees]
         contributors.extend(a)
```
Comment (mukundansundar) [inline .github/scripts/generate_release_notes.py:119]: check if `a` is not empty, if so log that it is empty?

**Review by yaron2 — APPROVED** [review]
lgtm

File: .github/scripts/generate_release_notes.py (hunk 1 above)
Comment (artursouza) [reply .github/scripts/generate_release_notes.py]: Fixed.

File: .github/scripts/generate_release_notes.py (hunk 2 above)
Comment (mukundansundar) [reply .github/scripts/generate_release_notes.py:119]: it is not a bug, but will help us track unassigned issues.

**Review by mukundansundar — DISMISSED** [review]

File: .github/scripts/generate_release_notes.py [hunk 3]
```diff
@@ -108,22 +109,28 @@ def get_repo_priority(name):
...
-        url = issueOrPR.html_url
     match = re.search(releaseNoteRegex, issueOrPR.body, re.M)
+    hasNote = False
     if match:
         repo = issueOrPR.repository
         note = match.group(1).strip()
         if note:
-            changes.append((repo, issueOrPR, note, contributors, url))
+            if note.upper() != "NOT APPLICABLE":
```
Comment (mukundansundar) [inline .github/scripts/generate_release_notes.py:126]: additionally check `N/A` ?

File: .github/scripts/generate_release_notes.py (hunk 2 above)
Comment (artursouza) [reply .github/scripts/generate_release_notes.py:121]: ok.

**Review by mukundansundar — APPROVED** [review]
lgtm

**Pull request merged**